viz_scripts/timeseries_analysis.ipynb

```diff
@@ -4,20 +4,16 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "These are the input parameters for the notebook. They will be automatically changed when the scripts to generate monthly statistics are run. You can modify them manually to generate multiple plots locally as well.\n",
-    "\n",
-    "Pass in `None` to remove the filters and plot all data. This is not recommended for production settings, but might be useful for reports based on data snapshots."
+    "## Generate Static Graphs"
    ]
   },
   {
-   "cell_type": "code",
-   "execution_count": null,
+   "cell_type": "markdown",
    "metadata": {},
-   "outputs": [],
    "source": [
-    "year = 2020\n",
-    "month = 11\n",
-    "program = \"prepilot\""
+    "These are the input parameters for the notebook. They will be automatically changed when the scripts to generate monthly statistics are run. You can modify them manually to generate multiple plots locally as well.\n",
+    "\n",
+    "Pass in `None` to remove the filters and plot all data. This is not recommended for production settings, but might be useful for reports based on data snapshots."
```

Comment:
Does this actually work? I thought that the code to autofill the parameters required that they be defined in the first cell of the notebook.

Reply:
So I did not write this, it was in all of the notebooks previously. If "None" is specified for date/program and run normally it does work with all dates/programs being loaded. So the parameter has to be defined, but it can be set to "None".

Reply:
I see that. Resolving this.